hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java

```diff
@@ -220,6 +220,14 @@ private SpanEventView.Builder generateViewBuilder(
     builder.setEndTimeMillis(event.getEndTimeMillis());
     builder.setDurationMillis(event.getEndTimeMillis() - event.getStartTimeMillis());
 
+    // internal duration
+    MetricValue metricValue =
+        event.getMetrics().getMetricMap().get(EnrichedSpanConstants.INTERNAL_SVC_LATENCY);
```

Comment:
Pls, use `getMetricValue` method of its parent class.

Reply:
Done.